packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -68,6 +68,39 @@ export interface DeploymentCircuitBreaker {
   readonly rollback?: boolean;
 }
 
+/**
+ * Alarm behavior of an ECS service
```

Comment:
Let's make this more descriptive: 
```suggestion
 * Deployment behavior when an ECS Service Deployment Alarm is triggered.
```

Reply:
ya did it